mix.exs

```diff
@@ -42,7 +42,8 @@ defmodule Phoenix.LiveDashboard.MixProject do
     [
       # Actual deps
       {:mime, "~> 1.6 or ~> 2.0"},
-      {:phoenix_live_view, "~> 0.17.7", phoenix_live_view_opts()},
+      {:phoenix, "~> 1.7.0-rc.0"},
+      {:phoenix_live_view, "~> 0.17.7 or ~> 0.18.0", phoenix_live_view_opts()},
```

Comment:
This is fine for now but not that our assets are precompiled, so once we actually release it, we should likely precompile them against 0.18.0 and require said version.